projects/1.20/assets/food-expansion-reimagined/foodexpansion/lang/zh_cn.json

```
@@ -0,0 +1,44 @@
+{
+    "item.foodexpansion.jelly": "翡翠果冻",
+    "item.foodexpansion.bacon": "培根",
+    "item.foodexpansion.cooked_bacon": "香脆培根",
+    "item.foodexpansion.fried_egg": "煎蛋",
+    "item.foodexpansion.bacon_and_egg": "培根煎蛋",
+    "item.foodexpansion.carrot_seed_soup": "胡萝卜汤",
+    "item.foodexpansion.squid": "生鱿鱼",
+    "item.foodexpansion.cooked_squid": "烤鱿鱼",
+    "item.foodexpansion.compressed_flesh": "腊肉",
+    "item.foodexpansion.chocolate_bar": "巧克力棒",
+    "item.foodexpansion.spider_soup": "蛛眼炖汤",
+    "item.foodexpansion.nether_wart_soup": "下界疣炖菜",
+    "item.foodexpansion.cactus_fruit": "仙人掌果",
+    "item.foodexpansion.horse_meat": "生马肉",
+    "item.foodexpansion.cooked_horse_meat": "熟马肉",
+    "item.foodexpansion.forbidden_fruit": "§e禁果",
+    "item.foodexpansion.cooked_mushroom": "烤蘑菇",
+    "item.foodexpansion.carrot_pie": "胡萝卜蛋糕",
+    "item.foodexpansion.bat_wing": "生蝙蝠翅膀",
+    "item.foodexpansion.cooked_bat_wing": "烤蝙蝠翅膀",
+    "item.foodexpansion.blaze_cream": "烈焰香奶汤",
+    "item.foodexpansion.melon_salad": "西瓜沙拉",
+    "item.foodexpansion.roasted_seed": "烤种子",
+    "item.foodexpansion.dough": "生面团",
+    "item.foodexpansion.wolf_meat": "生狼肉",
+    "item.foodexpansion.cooked_wolf_meat": "熟狼肉",
+    "item.foodexpansion.ocelot_meat": "生豹猫肉",
+    "item.foodexpansion.cooked_ocelot_meat": "熟豹猫肉",
+    "item.foodexpansion.starving_fruit": "§9饥饿之果",
+    "item.foodexpansion.lollipop": "棒棒糖",
+    "item.foodexpansion.beetroot_noodles": "甜菜面",
+    "item.foodexpansion.parrot_meat": "生鹦鹉肉",
+    "item.foodexpansion.cooked_parrot_meat": "熟鹦鹉肉",
+    "item.foodexpansion.llama_meat": "生羊驼肉",
+    "item.foodexpansion.cooked_llama_meat": "熟羊驼肉",
+    "item.foodexpansion.polar_bear_meat": "生北极熊肉",
+    "item.foodexpansion.cooked_polar_bear_meat": "熟北极熊肉",
+    "item.foodexpansion.veggie_stew": "蔬菜汤",
+    "item.foodexpansion.bat_soup": "蝙蝠浓汤",
+    "item.foodexpansion.golden_feast": "黄金盛餐",
```

```suggestion
    "item.foodexpansion.golden_feast": "黄金盛宴",
```
```suggestion
    "item.foodexpansion.golden_feast": "黄金大餐",
```
`盛餐`……总感觉不如`盛宴`常用
仅建议，可以不用改